doc/src/docs/InputOutputReference/01d-InputOutputReference.md

```diff
@@ -21687,19 +21687,19 @@ This is the maximum temperature of cooling water inlet or outlet that can occur
 
 #### Field: Electrical Efficiency Curve Name
 
-This is the name of Curve:Triquadratic object that defines the steady-state net electrical efficiency.   The electrical efficiency, <span>\({\eta_e}\)</span>, is a function of   the cooling water mass flow rate, <span>\({\dot m_{cw}}\)</span>, the temperature of the cooling water at the inlet, <span>\({T_{cw}}\)</span>, the steady-state net electrical power produced, <span>\({P_{net,ss}}\)</span>.
+This is the name of Curve:Triquadratic object that defines the steady-state net electrical efficiency.   The electrical efficiency, $\eta_e$, is a function of   the cooling water mass flow rate, $\dot m_{cw}$, the temperature of the cooling water at the inlet, $T_{cw}$, the steady-state net electrical power produced, $P_{net,ss}$.
 
-           <span>\({\eta_e} = f\left( {{P_{net,ss}},{{\dot m}_{cw}},{T_{cw}}} \right)\)</span>
+$${\eta_e} = f\left( {{P_{net,ss}},{{\dot m}_{cw}},{T_{cw}}} \right)$$
 
```

Comment:
@rglazer2 In my md editor, some of the equations are rendering better now, but the subscripts etc. in the text are not yet.  There's a reason for using the span tags.  @Myoldmopar can you please advise here?  If you look at the v8.3.0 pdf I/O Ref section for Generator:MicroCHP, the equations and other equation-like snippets in the text are not rendering.  The changes in this vicinity are intended to correct that.